opentech/apply/funds/views.py

```diff
@@ -959,3 +959,9 @@ def handle_no_permission(self):
         if self.raise_exception or self.request.user.is_authenticated:
             raise PermissionDenied(self.get_permission_denied_message())
         return redirect_to_login(self.request.get_full_path(), self.get_login_url(), self.get_redirect_field_name())
+
+
+@method_decorator(staff_required, name='dispatch')
+class SubmissionDetailSimplifiedView(DetailView):
```

Comment:
as per comment in template - please 404(3?) this page when there is no project associated.

Reply:
Fixed and rebased.